language/message/fr_FR.po

```diff
@@ -84,945 +84,992 @@ msgstr "Toilettes partiellement accessibles en fauteuil roulant"
+msgstr "jusqu’à {nextTransitionTime}"
+
+#: public/build/javascript/bundle.js:223
 msgid "string"
 msgstr ""
 
-#: public/build/javascript/bundle.js:1833
+#: public/build/javascript/bundle.js:13672
+#: public/build/javascript/bundle.js:15437
+msgid "Beers"
+msgstr "Bières pression"
+
+#: public/build/javascript/bundle.js:13773
+#: public/build/javascript/bundle.js:15577
+msgid "Internet access"
+msgstr "Accès internet"
+
+#: public/build/javascript/bundle.js:13773
+#: public/build/javascript/bundle.js:15577
+msgid "WiFi"
+msgstr "WiFi"
+
+#: public/build/javascript/bundle.js:15271
 msgid "Favorite places"
 msgstr "Lieux favoris"
 
-#: public/build/javascript/bundle.js:1833
+#: public/build/javascript/bundle.js:15271
 msgid "You have no favorite places. <br>You can add one by clicking on a place"
 msgstr ""
 "Vous n'avez pas encore de lieux favoris.<br>Ajoutez-en en cliquant sur un "
 "lieu"
 
-#: public/build/javascript/bundle.js:1833
+#: public/build/javascript/bundle.js:15273
 msgid "MY FAVORITES"
 msgstr "MES FAVORIS"
 
-#: public/build/javascript/bundle.js:1833
+#: public/build/javascript/bundle.js:15293
 msgid "Share"
 msgstr "Partager"
 
-#: public/build/javascript/bundle.js:1833
+#: public/build/javascript/bundle.js:15296
 msgid "Delete"
 msgstr "Supprimer"
 
-#: public/build/javascript/bundle.js:2503
+#: public/build/javascript/bundle.js:15322
 msgid "filter"
 msgstr "filtre"
 
-#: public/build/javascript/bundle.js:2547
-msgid "Register masq"
-msgstr "S'enregistrer sur Masq"
+#: public/build/javascript/bundle.js:15354
+msgid "Géolocation not available."
+msgstr "Géolocalisation impossible"
 
-#: public/build/javascript/bundle.js:2547
-msgid "We use MASQ to keep your preference data safe"
-msgstr "Nous utilisons MASQ pour conserver vos préférences en sécurité"
+#: public/build/javascript/bundle.js:15354
+msgid "You did not allow Qwant Maps to access your position."
+msgstr "vous n'avez pas autorisé Qwant Maps à accéder à votre position."
 
-#: public/build/javascript/bundle.js:2547
-msgid "Register"
-msgstr "S'enregistrer"
+#: public/build/javascript/bundle.js:15354
+msgid "ok"
+msgstr "ok"
 
-#: public/build/javascript/bundle.js:2556
-msgid "Back to favorite"
-msgstr "Retour aux favoris"
+#: public/build/javascript/bundle.js:15370
+msgid "Geolocation with privacy!"
+msgstr "La géolocalisation respectueuse de votre vie privée !"
 
-#: public/build/javascript/bundle.js:2556
-msgid "FAVORITES"
-msgstr "FAVORIS"
+#: public/build/javascript/bundle.js:15370
+#, python-brace-format
+msgid ""
+"You need to know where you are? Please allow your application to access your "
+"location so that it can tell you. As stated in our {privacyPolicyLink} "
+"privacy policy {closeTag}, Qwant will not store your geolocation "
+"information. We don't want to know your whereabouts."
+msgstr ""
+"Besoin de savoir où vous êtes ? Veuillez svp autoriser votre application à "
+"accéder à votre localisation pour qu’elle puisse vous l’indiquer.\n"
```

Comment:
the newline `\n` is not properly handled by the template. The letter "n" is rendered.

Reply:
Thank you, I removed it!